lib/bios/menus.robot

```diff
@@ -403,6 +403,14 @@ Reenter Menu
         Press Key N Times    1    ${ENTER}
     END
 
+Reload Dasharo Submenu
+    Press Key N Times    1    ${ESC}
+    Sleep    1s
+    Press Key N Times    1    ${ENTER}
+    Sleep    1s
+    ${submenu}=    Get Submenu Construction
+    RETURN    ${submenu}
+
```

Comment:
@Psotas 
See here: https://github.com/Dasharo/open-source-firmware-validation/pull/71/commits/a7a49babfa9a747247a432733972eaec22831ca6

Reply:
In this MR https://github.com/Dasharo/open-source-firmware-validation/pull/71/files this test progresses further, but in QEMU it seems it does not work: https://github.com/Dasharo/open-source-firmware-validation/pull/71/files

This is tracked here: https://github.com/Dasharo/dasharo-issues/issues/538

Please focus on other tests right now, not Secure Boot

Reply:
@Psotas I have pushed some more commits to this branch. Do you see anything more left than Secure Boot and this issue: https://github.com/Dasharo/open-source-firmware-validation/pull/55#discussion_r1364168586 ?

Reply:
@macpijan 
```
USH001.001 UEFI Shell
    [Documentation]    Check whether the DUT has the ability to boot into an
    ...    integrated UEFI Shell application.
    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    USH001.001 not supported
    Skip If    not ${UEFI_SHELL_SUPPORT}    USH001.001 not supported
    Power On
    Enter Boot Menu Tianocore
    Enter UEFI Shell Tianocore
    Read From Terminal Until    UEFI Interactive Shell
 ```
Currently, it enter to daharo menu.
Also `reset-to-defaults.robot` navigates menus incorrectly.

Reply:
@Psotas Reset to defaults has been reworked and passed in QEMU. Please confirm. It was done here: https://github.com/Dasharo/open-source-firmware-validation/pull/55/commits/52c01d50ce47145be3fc03d66814f3ee12ea187f

Reply:
@Psotas Where does this `USH001.001 UEFI Shell` code come from? On this branch it is already improved since 19th October.

Reply:
@macpijan 
`USH001.001 UEFI Shell` PASS, I'm not sure why it didn't work for me once, I'll keep an eye on it.

Testing `reset-to-defaults.robot`:
For testing, I had to turn off the part starting from `Reset To Defaults Tianocore` because it reset my connection after the series.

[reset-to-defaults_log.zip](https://github.com/Dasharo/open-source-firmware-validation/files/13186415/reset-to-defaults_log.zip)

In the `RTD005.001 F9 resets Early boot DMA Protection to true` test, there was once a problem with the correct reading of the menu structure.
It seems that `Enter Setup Menu Tianocore And Return Construction` read the "Second column", it added one line in the reading and `Get Index Of Matching Option In Menu` returned us 4 instead of 3.

```
Return: ['Select Language <Standard English> This is the option',
  'one adjusts to change',
  '>User Password Management the language for the',
  '>Device Manager current system',
  '> Dasharo System Features',
  '>One Time Boot',
  '>Boot Maintenance Manager',
  'Continue',
  'Reset']
```


`RTD011.001 F9 resets Watchdog timeout value to 500` test enter to "set value" but did not start entering any value.

`RTD012.001 F9 resets Fan profile to Silent` and `RTD013.001 F9 resets Platform sleep type to Suspend to Idle`
now when we try `Set Option State` but this option don't exist, we get FAIL.

Reply:
@macpijan 
The keyword `Get Setup SubMenu Construction` does not exist.
Can you remove where it appears and fix `uefi-password.robot`?

Reply:
@Psotas 

> Testing reset-to-defaults.robot:
For testing, I had to turn off the part starting from Reset To Defaults Tianocore because it reset connection 

Yes. That is why I run this test on QEMU. On MSI currently it cannot be executed.
Please check your JIRA notifications. Unless QEMU also does not work for you, then you should use updated image.

> The keyword Get Setup SubMenu Construction does not exist.
Can you remove where it appears and fix uefi-password.robo

I asked you to fix this in JIRA yesterday.

Reply:
DONE = remove kwds: Get Setup SubMenu Construction 
https://github.com/Dasharo/open-source-firmware-validation/pull/55/commits/ecf09723d15880268da63b981f390ad6575f6854